project_euler/problem_04/sol1.py

```diff
@@ -12,6 +12,10 @@ def solution(n):
     """Returns the largest palindrome made from the product of two 3-digit
     numbers which is less than n.
 
+    >>> solution(10000)
+    Traceback (most recent call last):
+    ...
+    ValueError: No such latest palindrome number.
```

Comment:
Nice!

Let's move this test below the others so we show how the function works before we show how it breaks,